yarn.lock

```diff
@@ -26899,12 +26899,7 @@ xtend@^4.0.0, xtend@^4.0.1, xtend@~4.0.1:
   resolved "https://registry.yarnpkg.com/xtend/-/xtend-4.0.2.tgz#bb72779f5fa465186b1f438f674fa347fdb5db54"
   integrity sha512-LKYU1iAXJXUgAXn9URjiu+MWhyUXHsvfp7mcuYm9dSUKK0/CjtrUwFAxD82/mCWbtLsGjFIad0wIsod4zrTAEQ==
 
-"y18n@^3.2.1 || ^4.0.0", y18n@^4.0.0:
-  version "4.0.0"
-  resolved "https://registry.yarnpkg.com/y18n/-/y18n-4.0.0.tgz#95ef94f85ecc81d007c264e190a120f0a3c8566b"
-  integrity sha512-r9S/ZyXu/Xu9q1tYlpsLIsa3EeLXXk0VwlxqTcFRfg9EhMW+17kbt9G0NrgCmhGb5vT2hyhJZLfDGx+7+5Uj/w==
-
-y18n@^5.0.5:
+y18n@5.0.5, "y18n@^3.2.1 || ^4.0.0", y18n@^4.0.0, y18n@^5.0.5:
```

Comment:
Does this major bump work ok across all the dependencies? Wondering what the breaking change was between 4 and 5. 🤔

Reply:
https://github.com/yargs/y18n/releases/tag/v5.0.0

Reply:
The `exports maps are now used, which modifies import behavior` can be an issue. I will have a look and see if I can upgrade the packages using the 4.x dependency.

Reply:
hmm, seems like some of the packages that has dependency on `y18n` have not yet updated to the `5.0.5` version. I will try to verify that this update don't have an negative effect on the project.

Reply:
```shell
yarn why y18n
yarn why v1.22.4
[1/4] 🤔  Why do we have the module "y18n"...?
[2/4] 🚚  Initialising dependency graph...
warning Resolution field "y18n@5.0.5" is incompatible with requested version "y18n@^4.0.0"
warning Resolution field "y18n@5.0.5" is incompatible with requested version "y18n@^4.0.0"
warning Resolution field "y18n@5.0.5" is incompatible with requested version "y18n@^4.0.0"
warning Resolution field "y18n@5.0.5" is incompatible with requested version "y18n@^4.0.0"
warning Resolution field "y18n@5.0.5" is incompatible with requested version "y18n@^4.0.0"
warning Resolution field "y18n@5.0.5" is incompatible with requested version "y18n@^3.2.1 || ^4.0.0"
warning Resolution field "y18n@5.0.5" is incompatible with requested version "y18n@^4.0.0"
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "y18n@5.0.5"
info Reasons this module exists
   - "_project_#cacache" depends on it
   - Hoisted from "_project_#cacache#y18n"
   - Hoisted from "_project_#yargs#y18n"
   - Hoisted from "_project_#webpack-dev-server#yargs#y18n"
   - Hoisted from "_project_#webpack-cli#yargs#y18n"
   - Hoisted from "_project_#rollup-plugin-visualizer#yargs#y18n"
   - Hoisted from "_project_#@grafana#toolkit#concurrently#yargs#y18n"
   - Hoisted from "_project_#lerna#@lerna#cli#yargs#y18n"
   - Hoisted from "_project_#jest#jest-cli#yargs#y18n"
   - Hoisted from "_project_#@grafana#ui#rollup-plugin-visualizer#yargs#y18n"
   - Hoisted from "_project_#jest#@jest#core#jest-runtime#yargs#y18n"
info Disk size without dependencies: "56KB"
info Disk size with unique dependencies: "56KB"
info Disk size with transitive dependencies: "56KB"
info Number of shared dependencies: 0
=> Found "@grafana/ui#y18n@5.0.5"
info Reasons this module exists
   - "_project_#@grafana#ui#@storybook#react#webpack#terser-webpack-plugin#cacache" depends on it
   - Hoisted from "_project_#@grafana#ui#@storybook#react#webpack#terser-webpack-plugin#cacache#y18n"
   - in the nohoist list ["/_project_/**/@types/*","/_project_/**/@types/*/**","/_project_/@storybook","/_project_/**/@storybook","/_project_/**/@storybook/**"]
info Disk size without dependencies: "56KB"
info Disk size with unique dependencies: "56KB"
info Disk size with transitive dependencies: "56KB"
info Number of shared dependencies: 0
✨  Done in 1.55s.

```

Reply:
So most of our dependencies have not upgraded to the latest version of y18n or yargs (that depends on y18n)

Reply:
I have verified the following:

- Tests run fine locally
- Applications starts and I can't find any weird console logs when navigating to the different parts.
- Storybook starts and works fine.